frontend/webapp/components/side.menu/menu/menu.styled.tsx

```diff
@@ -1,15 +1,16 @@
 import styled from 'styled-components';
 
-export const MenuContainer = styled.div`
+export const MenuContainer = styled.div<{ $isExpanded?: boolean; }>`
   width: 70px;
   transition: width 0.1s;
   height: 100%;
   border-right: 1px solid rgba(255, 255, 255, 0.04);
   background: ${({ theme }) => theme.colors.dark_blue};
-  &:hover {
+  ${ props  => props.$isExpanded && `
```

Comment:
@HeetVekariya you can pass it directly to the width attr like this: 
`width: ${({$isExoanded}) => $isExoanded ? 234 : 70}px`

Reply:
Made changes accordingly & tested the code